src/sidebar/styles-dark.css

```diff
@@ -25,10 +25,11 @@ body {
 /* material-design-lite (MDL) menu */
 #context-menu-button {
   background-image: url('dark-svg/three-dots-dark.svg');
+  background-color: #323744;
 }
 
 #footer-buttons.warning #context-menu-button {
-  background-color: rgba(0, 0, 0, 0.5) !important;
+  /*background-color: rgba(0, 0, 0, 0.5) !important;*/
```

Comment:
How does it works with warnings? Could you show us with a screenshot? (You can try to login to get a warning footbar)

Reply:
how can I produce a warning ? Any ideas

Reply:
Something like this maybe ?

![fornotessss](https://user-images.githubusercontent.com/25258877/34684719-0a5f858e-f4cc-11e7-9424-bf07b07af85a.gif)

Reply:
I guess it shouldn't be dark on the dark theme but the same as the light theme what do you think?

Reply:
Yeah you're right
I'll make changes and push that rightaway

Reply:
![yoyoyoyo](https://user-images.githubusercontent.com/25258877/34686365-4e407d62-f4d1-11e7-86bc-5845ac40c5a5.gif)